src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -364,6 +364,36 @@ public async Task<T> MakeHttpGetRequestWithFullPath<T>(string fullPath, string q
             return geoMasterResponse;
         }
 
+        /// <summary>
+        /// Gets the container logs for a site as a string
+        /// </summary>
+        /// <param name="subscriptionId">Subscription Id for the resource</param>
+        /// <param name="resourceGroupName">The resource group that the resource is part of </param>
+        /// <param name="name">Name of the resource</param>
+        /// 
+        /// <example>
+        /// The below example shows how you call <see cref="GetContainerLogs"/> to get container logs for this app.
+        /// <code>
+        /// public async static Task<![CDATA[<Response>]]> Run(DataProviders dp, OperationContext<![CDATA[<App>]]> cxt, Response res)
+        /// {
+        ///     var subId = cxt.Resource.SubscriptionId;
+        ///     var rg = cxt.Resource.ResourceGroup;
+        ///     var name = cxt.Resource.Name;
+        ///     
+        ///     string containerLogs = await dp.GeoMaster.GetContainerLogs(subId, rg, name);
+        ///         
+        ///     // do any processing on the string variable containerLogs
+        /// }
+        /// </code>
+        /// </example>
+        /// <returns></returns>
+        public async Task<string> GetContainerLogs(string subscriptionId, string resourceGroupName, string name)
```

Comment:
(This is probably true of all methods).
Recently, @hforeste did a change to runtime to separate out site name with slot name and pass it in Context resource.

Does this method "SitePathUtility.GetSitePath()" take into consideration slotName also? or parsing 'app(slot)' perhaps

Reply:
Yup it does. It allows passing site+slot as siteName(slot)